src/provisioner/provisioner.py

```diff
@@ -149,7 +149,7 @@ def apply_cluster_config(cortx_conf, cortx_release):
                 for node in storage_set['nodes']:
                     node_type = node['type']
                     node = dict(node_map[node_type], **node)
-                    if node['type'] == 'storage_node':
+                    if node['type'] in ['storage_node', 'data_node']:
```

Comment:
We dont have to hard code the names / types of the nodes. 
Type can be anything e.g. "test_node". We need to be agnostic to the names of the types.